### ISSUE TITLE
simplify error

### ISSUE THREAD

**Issue by hst-mdg** [josdejong/mathjs#998]
The following code
```
var t="-(x-y)";
var s=math.simplify(t);
console.log("t: "+t);
console.log("s: "+s.toString()); 
```                                                                                                                          
results in the output:
```
t: -(x-y)
s: -x
```
So something seems to be seriously wrong with the simplification rules?!
Or does simplify not work with multiple variables?!

(I am using https://cdnjs.cloudflare.com/ajax/libs/mathjs/3.18.0/math.min.js)

**Comment by firepick1** [comment]
Thanks for finding this bug. It was a special case in unary minus that did not handle  subtracted sub-expression.  Jos, I submitted changelist with fix and new unit test. All tests pass.
https://github.com/josdejong/mathjs/commit/f1d0beb2528b66b8f99a1d8a81ebf66d21e6ffca

**Comment by firepick1** [comment]
Reverted package-lock.json change in https://github.com/josdejong/mathjs/commit/9365c730fc34f64657e35bae28d7f44d6d8c44d7

**Comment by josdejong** [comment]
:+1: thanks for the quick fix @firepick1 . Will do a release tonight.

**Comment by josdejong** [comment]
I've released `v3.18.1` containing this fix.
